Sources/Kommunicate/Classes/KMConversationListViewController.swift

```diff
@@ -478,7 +478,32 @@ public class KMConversationListViewController: ALKBaseViewController, Localizabl
     private func createConversationAndLaunch() {
```

Comment:
Is this method used only by "Start new conversation" button?

Reply:
yes